src/manifest/pixiManifest.ts

```diff
@@ -118,14 +122,17 @@ function collectAssets(
     }
 
     const bundleAssets = localBundle.assets;
-
     const finalAssets = asset.getFinalTransformedChildren();
 
-    if (asset.transformChildren.length > 0)
+    if (asset.transformChildren.length > 0 && !asset.inheritedMetaData[options.tags!.mIgnore!])
```

Comment:
I guess this is kind of an early out, but wandering if we need this check? as we are filtering just below anyways?